cmd/atlas/internal/cmdapi/migrate.go

```diff
@@ -1645,9 +1779,8 @@ func isURL(s string) bool {
 
 // cmdEnvsRun executes a given command on each of the configured environment.
 func cmdEnvsRun[F any](
-	run func(*cobra.Command, []string, F) error,
-	set func(*cobra.Command, *Env) error,
-	cmd *cobra.Command, args []string, flags *F,
+	flags *F, setFlags func(*cobra.Command, *Env) error,
+	cmd *cobra.Command, runCmd func(*Env) error,
```

Comment:
```suggestion
	flags *F, 
	setFlags func(*cobra.Command, *Env) error,
	cmd *cobra.Command, 
	runCmd func(*Env) error,
```
Prefer this for readability

Reply:
Fixed